README.md

```diff
@@ -4,7 +4,7 @@ Audio Super Resolution Using Neural Networks
 This repository implements the audio super-resolution model proposed in:
 
 ```
-V. Kuleshov, Z. Enam, and S. Ermon. Audio Super Resolution Using Neural Networks. ICLR 2017 (Workshop track)
```

Comment:
Could you list the neurips paper the iclr 2017 paper here?

Reply:
@kuleshov done